procedures/pulpcore/update.md.erb

```diff
@@ -17,6 +17,14 @@ podman run --rm --volume $(pwd)/foreman-packaging/:/app/foreman-packaging/:Z  --
 
 The `python-pulp_2to3_migration` package is named differently due to a tito bug (the PyPI name uses dashes, not underscores). Due to this, the regular process of updating it cannot be used.
 
-* [ ] Have a regular `foreman-packaging` clone
+* [ ] Have a regular `pulpcore-packaging` clone
 * [ ] Run `obal update python-pulp_2to3_migration --version X.Y.Z`
 * [ ] As `obal` does not update dependencies, verify those are in sync with `setup.py`
+
+# Update to a new pulpcore release
+
+* [ ] Create a new `rpm/3.X` branch in `pulpcore-packaging` by using the previous release branch as a base
```

Comment:
In others we have parameters, so you can generate the right content. There should be context available:
https://github.com/theforeman/tool_belt/blob/8569eff9750ad953247b3ffb34c6d0bf01ff746e/lib/tool_belt/commands/procedure.rb#L12-L15

It may need some modification for pulpcore, but essentially if you now call it with `3.6` you can update this line
```suggestion
* [ ] Create a new `rpm/<%= develop %>` branch in `pulpcore-packaging` by using the previous release branch as a base: `git checkout -b rpm/<%= develop %> rpm/<%= release %>`
```

Obviously I didn't test this